README.md

```diff
@@ -157,12 +62,6 @@ Material Components for iOS uses
 copyright Google Inc. and licensed under
 [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).
 
-Several components use
-[MDFTextAccessibility](https://github.com/material-foundation/material-text-accessibility-ios),
-copyright Google Inc. and licensed under
-[Apache 2.0](https://github.com/material-foundation/material-text-accessibility-ios/blob/master/LICENSE)
-without a NOTICE file.
```

Comment:
We do not need to attribute dependencies that are pulled in via CocoaPods - CocoaPods will do that for apps.

Reply:
Won't Cocoapods pull in the Roboto down below as well? Why is this one special?

Reply:
I'm not certain what the expectations for Roboto's attributions are as a font, so I've left as-is. Can explore whether we need to attribute it in a follow-up change.

Reply:
So the lawyer cats signed-off on removing the MDF stuff but not yet the Roboto stuff?

Reply:
The lawyer cats never signed on or off of adding the CocoaPods attributions to our README.md. This attribution was added in error.

Reply:
+ajsecord@ who added that section explicitly in commit without any other changes.

Reply:
I've reverted this part of the change so that we're not blocking this PR.